manager/src/main/java/com/redhat/service/bridge/manager/api/user/ProcessorsAPI.java

```diff
@@ -77,19 +63,9 @@ public Response getProcessor(@NotEmpty @PathParam("bridgeId") String bridgeId, @
 
     @GET
     @Path("{bridgeId}/processors")
-    public Response listProcessors(@NotEmpty @PathParam("bridgeId") String bridgeId, @DefaultValue(PAGE_DEFAULT) @Min(PAGE_MIN) @QueryParam(PAGE) int page,
-            @DefaultValue(SIZE_DEFAULT) @Min(SIZE_MIN) @Max(SIZE_MAX) @QueryParam(PAGE_SIZE) int pageSize) {
-
-        String customerId = customerIdResolver.resolveCustomerId(identity.getPrincipal());
-        ListResult<Processor> processors = processorService.getProcessors(bridgeId, customerId, page, pageSize);
-        List<ProcessorResponse> px = processors.getItems().stream().map(processorService::toResponse).collect(toList());
-
-        ProcessorListResponse listResponse = new ProcessorListResponse();
-        listResponse.setItems(px);
-        listResponse.setPage(processors.getPage());
-        listResponse.setSize(processors.getSize());
-        listResponse.setTotal(processors.getTotal());
-        return Response.ok(listResponse).build();
+    public Response listProcessors(@NotEmpty @PathParam("bridgeId") String bridgeId, @BeanParam QueryInfo queryInfo) {
```

Comment:
Can you add a `@Valid` next to the `@BeanParam` to ensure it is validated.

Reply:
sure good point thanks